README.md

```diff
@@ -1,51 +1,116 @@
-# virtualbox-snapshotter
-Python script to automatize the Virtualbox snapshot process
+# VirtualBox Snapshotter
+
+![shield-license](https://img.shields.io/github/license/Meru3m/virtualbox-snapshotter)
+![shield-code-size](https://img.shields.io/github/languages/code-size/Meru3m/virtualbox-snapshotter)
+![shield-issues](https://img.shields.io/github/issues/Meru3m/virtualbox-snapshotter)
+
+## Description
+
+Python script to automate Virtualbox snapshotting process. It uses circular logic aka "oldest snapshot out, newest snapshot in" with an ability to customise total amount of snapshots to retain. Moreover, it provides and ability to customise snapshot details.
+
+## Dependency
+
+In order to function, this script relies on [VirtualBox SDK](https://www.virtualbox.org/wiki/Downloads)
 
 ## Installation process
-Youtube Guide: https://www.youtube.com/watch?v=uDEFQyu4MvM
- 
-### Install
-In order to function, this script requires the following step to be performed:
 
-**1. [Install the VirtualBox SDK](VirtualBox-SDK/README.md)** 
+__WARNING__ This guide shows how to install modules, required by this script, into ROOT of the Python. Usually, this is not a best practice as it may cause module conflicts in a long run. Please use Python [virtual environment (venv)](<https://docs.python.org/3/library/venv.html>) for a best practice. To avoid making this readme enormous, its up to a user to sort out the virtual environment for Python.
```

Comment:
recently I was thinking to replace the images and maybe create an updated guide. 
Thank you for pointing this, I'll try to show the installation in a venv, so that the process is cleaner.